VGowthamTheja/ecommerce-with-nextjs
# feat: Refactored authorization code and optimized it

## Description
---
- Removed the `/login` and `/signup` paths, and refactored them into a single path
- Designed the new path into the middleware
- Refurbished the UI/UX to match the latest trends

## Manual Testing
---
- Navigate to `/authorize` page upon landing on SideNavMenu.
- Check to whether the authorization functionality works as expected.

## Computed Devices
---
- [x] Chrome, Ubuntu,
- [x] Chrome, MacOS
